### PR TITLE
Added more links to projects

### PR DESCRIPTION
**Pull request by ShanTulshi**
without font-awesome as a dep!

### CODE REVIEW

File: _projects/README.md [hunk 1]
```diff
@@ -6,9 +6,11 @@ In the `_projects` folder, create a markdown file for the project. The frontmatt
 - `custom-css: project`
 - `title`
 - `desc` -- a couple of short lines to be rendered in the project card.
-- `github` -- optional -- link to GitHub repo containing the project.
 - `image` -- with a path to the project picture, which you should place in the `/img` folder.
```
Comment (andrewipark) [inline _projects/README.md]: Favor de borrar "with"

File: _projects/README.md [hunk 2]
```diff
@@ -6,9 +6,11 @@ In the `_projects` folder, create a markdown file for the project. The frontmatt
...
+- `github` -- optional -- link to GitHub repo containing the project.
 - `youtube` -- optional -- Video ID for YouTube video about the project. Video ID is the `v=` query parameter in a YouTube URL. 
 For example, for the video `https://www.youtube.com/watch?v=dQw4w9WgXcQ`, the video ID is `dQw4w9WgXcQ`.
+- `link` -- optional -- Link to the project site.
+- `dl` -- optional -- Link to project download.
```
Comment (andrewipark) [inline _projects/README.md]: descarga de [????] es una programa ya compilado y empieza Unity Player, o el `.unitypackage` del proyecto, o ...?

File: _projects/sample_project.md [hunk 3]
```diff
@@ -7,6 +7,8 @@ desc: |
   A mind-bending puzzle game where space and gravity themselves warp!
 github: https://github.com/chadfranzen/MovesLikeEscher
 youtube: AOTgxCqxBow
+link: http://localhost:4000
```
[the remark this answers is not in the document]
Comment (ShanTulshi) [reply _projects/sample_project.md]: I meant an external site, if applicable. But yes, letting `localhost:4000` go to prod is probably a bad idea.

File: _projects/README.md (hunk 2 above)
Comment (ShanTulshi) [reply _projects/README.md]: TBD, I requested that the link be to a `.zip` containing the exe

**Pull request merged**